packages/riipen-ui/src/components/Chip.jsx

```diff
@@ -68,103 +99,239 @@ class Chip extends React.Component {
       color,
       component: Component,
       disabled,
-      label,
-      onDelete,
+      hover,
+      iconStart: IconStart,
+      onIconClick,
+      onClick,
+      spacing,
       size,
       variant
     } = this.props;
 
+    const label = this.props.label || this.props.children;
+
     const theme = this.context;
 
+    const clickable = !!onClick;
+    const iconClickable = !!onIconClick;
+
     const className = clsx(
       "root",
       color,
       disabled ? "disabled" : null,
       size,
+      { hover },
       variant,
-      classes
+      classes,
+      { clickable }
     );
 
+    const handleClick = () => {
+      if (onClick) onClick();
+    };
+
     return (
       <React.Fragment>
-        <Component className={className}>
-          <span className={clsx("label")}>{label}</span>
-          {onDelete && (
-            <span className={clsx("delete", `delete-${color}`)}>
-              <FontAwesomeIcon fixedWidth icon={faTimes} onClick={onDelete} />
+        <Component onClick={handleClick} className={className}>
+          {IconStart && (
+            <span
+              onClick={onIconClick}
+              className={clsx("icon", "icon-start", {
+                iconClickable
+              })}
+            >
+              <IconStart />
             </span>
           )}
+          <span className={clsx("label")}>{label}</span>
         </Component>
         <style jsx>{`
           .root {
-            align-items: center;
             background-color: ${theme.palette.grey[300]};
-            border-radius: 16px;
-            border: none;
+            border-radius: 50px;
+            border: 1px solid;
+            border-color: transparent;
             box-sizing: border-box;
+            color: ${theme.palette.text.secondary};
+            cursor: default;
             display: inline-flex;
             font-family: ${theme.typography.fontFamily};
             font-size: 13px;
-            height: 32px;
             justify-content: center;
+            ${spacing !== 0 && `margin-bottom: ${theme.spacing(spacing)}px;`}
+            ${spacing !== 0 &&
+              `margin-right: ${theme.spacing(
+                spacing
+              )}px;`}
```

Comment:
This allows the user to overwrite the margins by providing spacing = 0

Reply:
All "base" UI elements shouldn't have margins (in most cases) by default. It should be up to the application to determine how to render the element if it needs external spacing. I think we should remove this margin code. It also would eliminate the need for the `spacing` prop which doesn't really fit the types of props we want in the UI kit.